sos/plugins/apache.py

```diff
@@ -65,18 +73,25 @@ def setup(self):
         self.add_copy_spec_limit("/var/log/httpd22/error_log", 5)
         self.add_copy_spec_limit("/var/log/httpd22/ssl_access_log", 5)
         self.add_copy_spec_limit("/var/log/httpd22/ssl_error_log", 5)
-        # Red Hat JBoss Web Server 3.x
+        # Red Hat JBoss Web Server 3.x and Red Hat Software Collections
         self.add_copy_spec_limit("/var/log/httpd24/access_log", 5)
         self.add_copy_spec_limit("/var/log/httpd24/error_log", 5)
         self.add_copy_spec_limit("/var/log/httpd24/ssl_access_log", 5)
         self.add_copy_spec_limit("/var/log/httpd24/ssl_error_log", 5)
+        # Red Hat JBoss Core Services
+        self.add_copy_spec_limit("/opt/rh/jbcs-httpd24/root/var/log/httpd/access_log", 5)
+        self.add_copy_spec_limit("/opt/rh/jbcs-httpd24/root/var/log/httpd/error_log", 5)
+        self.add_copy_spec_limit("/opt/rh/jbcs-httpd24/root/var/log/httpd/ssl_access_log", 5)
+        self.add_copy_spec_limit("/opt/rh/jbcs-httpd24/root/var/log/httpd/ssl_error_log", 5)
         if self.get_option("log"):
             self.add_copy_spec([
                 "/var/log/httpd/*",
                 # JBoss Enterprise Web Server 2.x
                 "/var/log/httpd22/*",
                 # Red Hat JBoss Web Server 3.x
-                "/var/log/httpd24/*"
+                "/var/log/httpd24/*",
+                # Red Hat JBoss Core Services
+                "/opt/rh/jbcs-httpd24/root/var/log/httpd/*"
```

Comment:
Couldn't this directory contain many big logfiles, causing sosreport to collect too much data? (anyway since this plugin collects whole `/var/log/httpd2[2,4]/` directories, I would expect the answer is no)